"Fix imports" reports

Hello,

The "Fix imports" reports seem offtopic for this linter and they are annoying.

Related to #14

Note: I'm using the current dev version of golangci-lint.

```go
package main

import (
	"fmt"
	"testing"
)

func TestName(t *testing.T) {
	data := "bar"
	s := fmt.Sprintf("foo %s", data)
	println(s)
}
```

```console
$ ./golangci-lint run --no-config --disable-all -Eperfsprint
main_test.go:10:7: fmt.Sprintf can be replaced with string addition (perfsprint)
        s := fmt.Sprintf("foo %s", data)
             ^
main_test.go:4:2: Fix imports (perfsprint)
        "fmt"
        ^
```

In this context, your linter has detected that `fmt` is only related to `fmt.Sprintf`, so you are suggesting to remove the import.
I understand why (because you want to "autofix" the problem), but the import management is not something that your linter should handle (at least inside golangci-lint) because other linters will handle that.

Another problem, if I want to ignore some reports then the reports `Fix imports` will always happen.

```yml
linters:
  disable-all: true
  enable:
    - perfsprint

issues:
  exclude:
    - 'fmt.Sprintf can be replaced with string addition'
```

```console
$ ./golangci-lint run
main_test.go:4:2: Fix imports (perfsprint)
        "fmt"
```

It's the same thing if I use a `nolint` directive (without `exclude` configuration):

```go
package main

import (
	"fmt"
	"testing"
)

func TestName(t *testing.T) {
	data := "bar"
	s := fmt.Sprintf("foo %s", data) //nolint:perfsprint
	println(s)
}
```

```console
$ ./golangci-lint run --no-config --disable-all -Eperfsprint
main_test.go:4:2: Fix imports (perfsprint)
        "fmt"
```

So I suggest removing those reports, at least for golangci-lint.


<!--
```yml
issues:
  exclude-use-default: false
  max-issues-per-linter: 0
  max-same-issues: 0
  exclude:
    - 'fmt.Sprintf can be replaced with string addition'
  exclude-rules:
    - path: '(.+)\.go'
      text: "Fix imports"
      linters:
        - perfsprint
```
-->

--- a/analyzer/analyzer.go
+++ b/analyzer/analyzer.go
@@ -17,18 +17,20 @@ import (
 )
 
 type perfSprint struct {
-	intConv  bool
-	errError bool
-	errorf   bool
-	sprintf1 bool
+	intConv    bool
+	errError   bool
+	errorf     bool
+	sprintf1   bool
+	fiximports bool
 }
 
 func newPerfSprint() *perfSprint {
 	return &perfSprint{
-		intConv:  true,
-		errError: false,
-		errorf:   true,
-		sprintf1: true,
+		intConv:    true,
+		errError:   false,
+		errorf:     true,
+		sprintf1:   true,
+		fiximports: true,
 	}
 }
 
@@ -44,6 +46,7 @@ func New() *analysis.Analyzer {
 	r.Flags.BoolVar(&n.errError, "err-error", false, "optimizes into err.Error() even if it is only equivalent for non-nil errors")
 	r.Flags.BoolVar(&n.errorf, "errorf", true, "optimizes fmt.Errorf")
 	r.Flags.BoolVar(&n.sprintf1, "sprintf1", true, "optimizes fmt.Sprintf with only one argument")
+	r.Flags.BoolVar(&n.fiximports, "fiximports", true, "fix needed imports from other fixes")
 	return r
 }
 
@@ -486,7 +489,7 @@ func (n *perfSprint) run(pass *analysis.Pass) (interface{}, error) {
 		}
 	})
 
-	if len(removedFmtUsages) > 0 {
+	if len(removedFmtUsages) > 0 && n.fiximports {
 		for _, pkg := range pass.Pkg.Imports() {
 			if pkg.Path() == "fmt" {
 				insp = pass.ResultOf[inspect.Analyzer].(*inspector.Inspector)

